R/measures.R

```diff
@@ -64,8 +68,13 @@
 #' }
 riem_measures <- function(
     station = "VOHY",
-    date_start = "2014-01-01",
-    date_end = as.character(Sys.Date())) {
+    data = "all",
+    elev = "no",
+    latlon = "yes",
```

Comment:
@JElchison as you prefer! But if we start changing the default value we'd need to use https://lifecycle.r-lib.org/articles/communicate.html Maybe this can be a separate issue?

Reply:
Summarized into #50